src/client/src/rt-platforms/browser/utils/sendNotification.ts

```diff
@@ -8,18 +8,31 @@ export interface NotificationMessage {
   tradeNotification: Trade
 }
 
+let registration: ServiceWorkerRegistration
+navigator.serviceWorker &&
+  navigator.serviceWorker.ready.then(reg => {
+    registration = reg
+  })
+
 export const sendNotification = ({ tradeNotification }: NotificationMessage) => {
   const status = tradeNotification.status === 'done' ? 'Accepted' : 'Rejected'
   const title = `Trade ${status}: ${tradeNotification.direction} ${tradeNotification.dealtCurrency} ${tradeNotification.notional}`
   const body = `vs. ${tradeNotification.termsCurrency} \nRate ${tradeNotification.spotRate}    Trade ID ${tradeNotification.tradeId}`
 
-  const options = {
-    body,
-    icon: './static/media/adaptive-logo-without-background.png',
+  const options: NotificationOptions = {
+    body: body,
+    icon: './static/media/reactive-trader.ico',
     dir: 'ltr',
+    requireInteraction: true,
+    actions: [{ action: 'highlight-trade', title: 'Highlight in blotter' }],
+    data: tradeNotification,
   }
 
-  /* eslint-disable @typescript-eslint/no-unused-vars */
-  // @ts-ignore
-  const notification = new Notification(title, options)
+  if (registration) {
+    registration.showNotification(title, options)
+  } else {
+    delete options.actions
```

Comment:
can this else be removed, seems to serve no purpose

Reply:
Instantiating `const notification = ... ` actually displays a notification, so that else block is a fallback mechanism to still display a notification (without blotter highlighting functionality) if the service worker doesn't register for whatever reason.